docs/development/guide.md

```diff
@@ -39,9 +39,8 @@ If you make any changes in the project structure, remember to update it.
 
 Use the following tools to develop the AsyncApi React component:
```

Comment:
I've never liked this section, because it's not like a somehow use react or TS cli to develop directly, they are dependencies and are downloaded automatically, I'm not bringing them myself, it's misleading

to be discussed with @kazydek and @magicmatatjahu

Reply:
I'm not insisting on keeping it if you think it is more of a hassle than help 🤷‍♀